cbits/cryptonite_aes.h

```diff
@@ -55,6 +55,18 @@ typedef struct {
 	uint64_t length_input;
 } aes_gcm;
 
+/* size = 80 */
+typedef struct {
+	aes_block xi;
+	aes_block header_cbcmac;
+	aes_block b0;
+	aes_block nonce;
+	unsigned length_aad;
+	unsigned length_input;
+        int      length_M;
+        int      length_L;
+} aes_ccm;
+
```

Comment:
I think it's best not to use types with variable size like `int` or `unsigned` but fixed-sized like `uint8_t` and `uint32_t`.

Reply:
@ocheron Many thanks for your through review I agree with your about the changes, also there're three asserts in the change I think they better be removed.